adam-core/src/main/scala/org/bdgenomics/adam/rdd/BroadcastRegionJoin.scala

```diff
@@ -165,7 +165,7 @@ object BroadcastRegionJoin extends RegionJoin {
  *
  * @param regions The input-set of regions.
  */
-private[rdd] class NonoverlappingRegions(regions: Iterable[ReferenceRegion]) extends Serializable {
```

Comment:
I mean, it is a model, right? Mostly just a container with an invariant (regions shouldn't overlap).